api-gateway/src/main/resources/static/scripts/app.js

```diff
@@ -10,9 +10,9 @@ const whiteList = new Set([
 /* App Module */
 const petClinicApp = angular.module('petClinicApp', [
     'ui.router', 'layoutNav', 'layoutFooter', 'layoutWelcome', 'ownerList', 'ownerDetails', 'ownerForm', 'ownerRegister', 'petRegister', 'petForm'
-    , 'visits', 'vetList','vetForm','vetDetails', 'visitList', 'billForm', 'billUpdateForm', 'loginForm', 'rolesDetails', 'signupForm',
-    'billDetails', 'billsByOwnerId', 'billHistory','billsByVetId','inventoryList', 'inventoryForm', 'productForm','inventoryProductList', 'inventoryUpdateForm', 'productUpdateForm'
-    , 'verification' , 'adminPanel','resetPwdForm','forgotPwdForm','petTypeList']);
+    , 'visits', 'vetList','vetForm','vetDetails', 'visitList', 'billForm', 'billUpdateForm', 'loginForm', 'rolesDetails', 'signupForm', 'productDetailsInfo',
```

Comment:
Why do you think it was necessary to add "productDetailsInfo" in the app.js?

Reply:
If i didn't add the product detail info, in the app.js, then my page would never render when called.

Reply:
That's very smart thank you for your input Mr Siddiqui.